### ISSUE TITLE
Auto focus search bar

### ISSUE THREAD

**Issue by AyanChoudhary** [creativecommons/cccatalog-frontend#1032]
## Problem

The search bar does not get auto focused when the page loads, also, most webpages with a search have a quick focus option on clicking `/`

## Description

We will have to enable auto-focus on the search when the page loads, it will have to be audited that on which pages we need it to auto-focus(for eg. it is necessary for the homepage but maybe not in the search-guide page), whereas the quick focus should be enabled in all the pages

## Implementation

- [ ] I would be interested in implementing this feature.

**Comment by abhisheknaiidu** [comment]
@AyanChoudhary I have made the changes, but don’t you think, autofocus seems to be an issue for accessibility? 
Ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefautofocus
